ethereum.md

```diff
@@ -17,14 +23,18 @@ module ETHEREUM-SIMULATION
     imports VERIFICATION
 ```
 
-```{.k .rvk}
+```{.k .rvk .standalone .node}
     imports K-REFLECTION
 ```
 
+```{.k .uiuck .rvk .node}
+    imports IELE-NODE
+```
+
 A IELE simulation is a list of IELE commands.
 Some IELE commands take a specification of IELE state (eg. for an account or transaction).
 
-```{.k .uiuck .rvk}
+```{.k .uiuck .rvk .standalone }
```

Comment:
There is no IELESimulation defined in .node version? but In .node version, the k cell in the configuration still accepts IELESimulation, right?